docs/code-search/queries/language.mdx

```diff
@@ -257,7 +257,7 @@ For example,
 
 ![patter-type](https://storage.googleapis.com/sourcegraph-assets/Docs/CS-LANG/pattern-type.png)
 
-Set whether the pattern should run a literal search, regular expression search, or structural search. This parameter is available as a command-line and accessibility option and is synonymous with the visual [search pattern](#search-pattern) toggles.
+Set whether the pattern should run a [keyword search](/code-search/queries#keyword-search), literal search, regular expression search, or structural search. This parameter is available as a command-line and accessibility option and is synonymous with the visual [search pattern](#search-pattern) toggles.
```

Comment:
I followed the link to `#search-pattern` and noticed that section needs updating too! Right now we say "By default, the pattern is searched literally." We can just mention keyword search there and not mention literal search at all.